### PR TITLE
Replace Travis by GitHub Actions

### PR DESCRIPTION
**Pull request by Avaq**
Just trying this out.

### CODE REVIEW

**Comment by dicearr** [comment]
The job logs are a bit messy specially because of the colors, other than that, is there any special reason for dropping Codecov?

**Comment by Avaq** [comment]
> any special reason for dropping Codecov?

I thought it wasn't working anymore. Turned out to be a misconfiguration. I've put Codecov back.

**Comment by Avaq** [comment]
If this works out well, I'm moving it to [`fluture-template`](https://github.com/fluture-js/fluture-template)

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 # [![Fluture](logo.png)](#butterfly)
 
 [![Build Status][]](https://travis-ci.org/fluture-js/Fluture)
-[![Code Coverage][]](https://codecov.io/gh/fluture-js/Fluture/branch/master)
```
Comment (dicearr) [inline README.md]: Shouldn't the badges be back?

**Pull request merged**